### ISSUE TITLE
Configure HA To Connect To Specific Exit Node

### ISSUE THREAD

**Issue by thelightningist** [hassio-addons/addon-tailscale#24]
I have a NAT'd home network that I can't change, I was attempting to use my raspberry pi 4 home assistant as a Nginx reverse proxy (that was already running and working with my previously publically accessible network configuration) and have an exit node on the VPS.

I have portainer and can sign into the console of the tailscale container but I'm struggling.

The problem I am running into is that once an exit node in Tailscale is created, you have to enter in a command for the other tailscale clients to forward traffic through it.

Specifically, I need to run the sudo tailscale up --exit-node=<vps-tailscale-ip> command on the home assistant add-on for tailscale. Any ideas? When I try executing these commands in the Portainer console, it acts like tailscale isn’t installed even though it is. 

Otherwise, would it be possible to create a configuration option to bind these to the tailscale IP address of another client configured as an exit node?

**Comment by brunoenten** [comment]
You can set the exit node in the addon configuration.

![image](https://user-images.githubusercontent.com/722037/132299614-d70d82f3-a3a4-437f-b4f1-345ce2664d5c.png)

**Comment by thelightningist** [comment]
> You can set the exit node in the addon configuration.
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/722037/132299614-d70d82f3-a3a4-437f-b4f1-345ce2664d5c.png)
> 
> 

Unfortunately this does not exist in the configuration panel of the official Tailscale add-on in Home Assistant. 

“This add-on has no additional configuration options for the add-on itself.”

https://github.com/hassio-addons/addon-tailscale/blob/main/tailscale/DOCS.md

The configuration panel you are showing, as best as I can tell, is from the HACS repository correct?

**Comment by brunoenten** [comment]
Yes, you are correct. Sorry I forgot I had to go through HACS to install the addon.

**Comment by thelightningist** [comment]
> Yes, you are correct. Sorry I forgot I had to go through HACS to install the addon.
> 
> It is the addon from tsujamin's repositories.

Maybe I should try it again. I did attempt to do this but for some reason it almost bricked my HA instance entirely. Maybe I should try removing the official one before attempting again?
